Sources/SwiftFormat/Pipelines+Generated.swift

```diff
@@ -243,8 +233,12 @@ extension LintPipeline {
     return .visitChildren
   }
 
-  func visit(_ node: SwitchStmtSyntax) -> SyntaxVisitorContinueKind {
+  func visit(_ node: SwitchCaseListSyntax) -> SyntaxVisitorContinueKind {
```

Comment:
Oh, crap, I never regenerated this, did I 😞 

At some point we should add a test that verifies that this file is up-to-date.

Reply:
Maybe a pre-commit hook? We might want to do the same for formatting the formatter soon too.

Reply:
Maybe a pre-commit hook? We might want to do the same for running the formatter on the formatter soon too.

Reply:
We could make a test that invokes `generate-pipeline` and then verifies that the output is the same as the file in the source tree. Then once we're on CI, failure to regenerate would catch it.

A pre-commit hook that just auto-generates it could be smoother since it would prevent failure, but I'm a little wary of executing that much logic in a pre-commit hook (I'm not sure if there's ever a time where we _wouldn't_ want that thing to run automatically).